thavalon-server/src/game/messages.rs

```diff
@@ -82,6 +82,9 @@ pub enum Message {
     /// Announces that players should submit votes for the latest proposal.
     CommenceVoting,
 
+    /// Announces that a player has submitted a valid vote.
+    VoteReceived,
```

Comment:
future extension: report who hasn't voted yet for public nagging (although that might leak role information)

Reply:
I think we don't want to publicly reveal names because of information leak is my concern.